extensions/flutter-pub-dev-search/src/index.tsx

```diff
@@ -0,0 +1,133 @@
+import {
+  ActionPanel,
+  CopyToClipboardAction,
+  List,
+  OpenInBrowserAction,
+  showToast,
+  ToastStyle,
+  randomId,
+} from "@raycast/api";
+import { useState, useEffect, useRef } from "react";
+import fetch, { AbortError } from "node-fetch";
+
+export default function Command() {
+  const { state, search } = useSearch();
+
+  return (
+    <List isLoading={state.isLoading} onSearchTextChange={search} searchBarPlaceholder="Search for a Dart/Flutter package..." throttle>
+      <List.Section title="Results" subtitle={state.results.length + ""}>
+        {state.results.map((searchResult) => (
+          <SearchListItem key={searchResult.id} searchResult={searchResult} />
+        ))}
+      </List.Section>
+    </List>
+  );
+}
+
+function SearchListItem({ searchResult }: { searchResult: SearchResult }) {
+  return (
+    <List.Item
+      title={searchResult.name}
+      // subtitle={searchResult.description}
+      // accessoryTitle={searchResult.version}
+      actions={
+        <ActionPanel>
+          <ActionPanel.Section>
+            <CopyToClipboardAction
+              title="Copy Install Command"
+              content={`flutter pub add ${searchResult.name}`}
+              onCopy={(content) => showToast( ToastStyle.Success,`${content} Copied to clipboard!\n Now go and paste the command into your IDE's terminal to add the package.`)}
```

Comment:
Showing another Toast in the `onCopy` handler won't work since it first shows the HUD-style toast, which hides the Raycast window, and then shows another Toast, which wouldn't be visible. To be consistent with other extensions, I'd suggest to just use the default `CopyToClipboardAction` and remove the `onCopy` handler.